client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx

```diff
@@ -68,6 +79,42 @@ export class DowngradeStep extends Component {
 						{ args: { amount } }
 				  );
 		}
+
+		if ( 'downgrade-monthly' === upsell ) {
+			refundTitle = translate(
+				'Would you rather switch to the more affordable monthly subscription?'
+			);
+
+			if ( cancelBundledDomain && includedDomainPurchase ) {
+				refundReason = (
+					<p>
+						{ translate(
+							'You will keep all features of your current plan including the domain %(domain)s but the plan period will be reduced.',
+							{ args: { domain: includedDomainPurchase.meta } }
+						) }
+					</p>
+				);
+			} else {
+				refundReason = (
+					<p>
+						{ translate(
+							'You will keep most of the features of your current plan but will not have a free domain.'
```

Comment:
Is "free domain" the terminology we typically use for this?  It's only free for the first year, and I thought we usually specify that.